charts/fission-all/templates/fluentbit.yaml

```diff
@@ -32,6 +32,14 @@ spec:
       labels:
         svc: logger
     spec:
+      initContainers:
```

Comment:
Is this really needed?  Is the change in logger.go not enough?

Reply:
Yes, because fluentbit also needs it.
`charts/fission-all/config/fluentbit.conf` Line 13
```
DB /var/log/fission/flb_kube.db
```